tdrs-backend/tdpservice/scheduling/tasks.py

```diff
@@ -0,0 +1,15 @@
+"""Shared celery tasks file for beat."""
+from __future__ import absolute_import
+from celery import shared_task
+import logging
+from .db_backup import run_backup
+
+logger = logging.getLogger(__name__)
+
+@shared_task
+def nightly_postgres(*args):
```

Comment:
Should probably rename this function as while that's the intent, the odd handling of args here could be refactored.